packages/p2p/src/components/modal-manager/modals/ad-created-modal/ad-created-modal.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button, Checkbox, Modal, Text } from '@deriv/components';
+import { observer } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
```

Comment:
If im not mistaken, we should be using the Localize and localize from `Components/i18next` as after talking with yashim, he had told me the process of how p2p do our translations is different than the rest of deriv-app. though i know importing from translations would fix some of our tslint issues, we should use `Components/i18next` 👍 
```suggestion
import { Localize } from 'Components/i18next';
```